### PR TITLE
driver_metrics config added

### PR DESCRIPTION
**Pull request by Kinju011**

### CODE REVIEW

**Comment by klobuczek** [comment]
@Kinju011 in Neo4j::Driver::Ext::ConfigConverter handle `driver_metrics` in similar way `encryption` is handled. Avoid any duplication.

**Comment by Kinju011** [comment]
> @Kinju011 in Neo4j::Driver::Ext::ConfigConverter handle `driver_metrics` in similar way `encryption` is handled. Avoid any duplication.

okay, thanks
will do that.

File: jruby/neo4j/driver/ext/config_converter.rb [hunk 1]
```diff
@@ -19,6 +19,9 @@ def config_method(key, value)
           when 'encryption'
             method = :without_encryption unless value
             value = nil
+          when 'driver_metrics'
+            method = :without_driver_metrics unless value
+            value = nil
```
Comment (klobuczek) [inline jruby/neo4j/driver/ext/config_converter.rb]: as I mentioned earlier this should be solved in a more DRY way

**Review by klobuczek — CHANGES_REQUESTED** [review]

File: jruby/neo4j/driver/ext/config_converter.rb (hunk 1 above)
Comment (Kinju011) [reply jruby/neo4j/driver/ext/config_converter.rb]: okay, updating it it.

**Review by klobuczek — APPROVED** [review]

**Pull request merged**